### PR TITLE
sql: add more states for ADMIN SHOW DDL

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 1]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
...
     - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `none` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
     - `running`：表示该操作正在执行。
     - `synced`：表示该操作已经执行成功，且所有 TiDB 实例都已经同步该状态。
     - `rollback done`：表示该操作执行失败，回滚完成。
     - `rollingback`：表示该操作执行失败，正在回滚。
     - `cancelling`：表示正在取消该操作。这个状态只有在用 [`ADMIN CANCEL DDL JOBS`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消 DDL 任务时才会出现。
+    - `cancelled`：表示该操作已经取消。
+    - `pausing`：表示正在暂停该操作。
     - `paused`：表示 DDL 已被暂停运行。这个状态只有在用 [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) 命令暂停 DDL 任务时才会出现。可以通过 [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) 命令进行恢复运行。
+    - `done`：表示该操作在 owner 节点已经执行成功，但其他 TiDB 实例还没有同步该状态，一般是由于未提交事务持有元数据锁导致阻塞了状态变更。
```
Comment (D3Hunter) [inline sql-statements/sql-statement-admin-show-ddl.md:91]:
```suggestion
    - `done`：表示该操作在 owner 节点已经执行成功，但其他 TiDB 实例还没有同步该 DDL 所执行的变更。
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 2]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
 - `START_TIME`：DDL 操作的开始时间。
 - `END_TIME`：DDL 操作的结束时间。
 - `STATE`：DDL 操作的状态。常见的状态有以下几种：
+    - `none`：表示该操作还未开始。
     - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `none` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:82]:
```suggestion
    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `done` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 3]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
 - `START_TIME`：DDL 操作的开始时间。
 - `END_TIME`：DDL 操作的结束时间。
 - `STATE`：DDL 操作的状态。常见的状态有以下几种：
-    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `none` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
+    - `none`：表示该操作还未开始。
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-show-ddl.md:81]:
```suggestion
    - `none`：表示该操作尚未开始。
```

**Review by pcqz — APPROVED** [review]

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 4]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
...
+    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `done` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-show-ddl.md:82]: "会变为 `done` 状态" 是指 "`queueing` 状态会变为 `done` 状态" 吗

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 5]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
...
     - `running`：表示该操作正在执行。
     - `synced`：表示该操作已经执行成功，且所有 TiDB 实例都已经同步该状态。
     - `rollback done`：表示该操作执行失败，回滚完成。
     - `rollingback`：表示该操作执行失败，正在回滚。
     - `cancelling`：表示正在取消该操作。这个状态只有在用 [`ADMIN CANCEL DDL JOBS`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消 DDL 任务时才会出现。
+    - `cancelled`：表示该操作已经取消。
+    - `pausing`：表示正在暂停该操作。
     - `paused`：表示 DDL 已被暂停运行。这个状态只有在用 [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) 命令暂停 DDL 任务时才会出现。可以通过 [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) 命令进行恢复运行。
+    - `done`：表示该操作在 owner 节点已经执行成功，但其他 TiDB 实例还没有同步该 DDL 任务所执行的变更。
```
Comment (qiancai) [inline sql-statements/sql-statement-admin-show-ddl.md:91]:
```suggestion
    - `done`：表示该操作在 TiDB owner 节点已经执行成功，但其他 TiDB 节点还没有同步该 DDL 任务所执行的变更。
```

File: sql-statements/sql-statement-admin-show-ddl.md (hunk 4 above)
Comment (hfxsd) [reply sql-statements/sql-statement-admin-show-ddl.md:82]: 是的

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 6]
```diff
@@ -78,13 +78,17 @@ OWNER_ADDRESS: 0.0.0.0:4000
 - `START_TIME`：DDL 操作的开始时间。
 - `END_TIME`：DDL 操作的结束时间。
 - `STATE`：DDL 操作的状态。常见的状态有以下几种：
-    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `none` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
+    - `none`：表示该操作尚未开始。
+    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，会变为 `done` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:82]:
```suggestion
    - `queueing`：表示该操作任务已经进入 DDL 任务队列中，但尚未执行，因为还在排队等待前面的 DDL 任务完成。另一种原因可能是执行 `DROP` 操作后，`queueing` 状态会变为 `done` 状态，但是很快会更新为 `synced` 状态，表示所有 TiDB 实例都已经同步到该状态。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**